### PR TITLE
[docs] update transifex doc link

### PR DESCRIPTION
**Pull request by mess110**
The old link is no longer working.

#13364

### CODE REVIEW

**Comment by Empact** [comment]
Why open a new PR? Better to keep the history together IMO. I recommend re-opening the prior PR.

**Comment by mess110** [comment]
When rebasing, I accidentally rewrote the commit previous to that. To keep history, I linked it.

File: doc/translation_process.md [hunk 1]
```diff
@@ -46,9 +46,7 @@ Visit the [Transifex Signup](https://www.transifex.com/signup/) page to create a
 You can find the Bitcoin translation project at [https://www.transifex.com/projects/p/bitcoin/](https://www.transifex.com/projects/p/bitcoin/).
 
 ### Installing the Transifex client command-line tool
-The client it used to fetch updated translations. If you are having problems, or need more details, see [http://docs.transifex.com/developer/client/setup](http://docs.transifex.com/developer/client/setup)
-
-**For Linux and Mac**
+The client it used to fetch updated translations. If you are having problems, or need more details, see [https://docs.transifex.com/client/installing-the-client](https://docs.transifex.com/client/installing-the-client)
```
Comment (fanquake) [inline doc/translation_process.md]: s/it/is

**Comment by fanquake** [comment]
utACK after fixing the typo. 
In future please don't open new PRs for the same changes.

**Comment by fanquake** [comment]
ACK 2b30ccc

**Comment by MarcoFalke** [comment]
ACK 2b30ccc

**Review by randolf — APPROVED** [review]

**Pull request merged**